lib/seed_helper.rb

```diff
@@ -87,6 +87,10 @@ def update_role_permissions(role, options)
           role.add_permissions(missing_permissions, :save! => true)
         end
 
+        # The built in role may have additional permissions added to it by users.
+        # To remove permissions from the default role use an explicit migration.
+        return if role.builtin == Role::BUILTIN_DEFAULT_ROLE
```

Comment:
We don't have a reliable way of telling if any extra permissions were done by the user or us (association audits will only work for 1.18+, and even then not sure if it will catch this correctly), I believe it is better to explicitly remove any permissions in migration than here.